### PR TITLE
8.0 ps 5736

### PR DESCRIPTION
**Pull request by gl-sergei**
No functional changes. Testcase changes:
- CREATE TEMPORARY TABLE t06 (a INT) ENGINE=InnoDB ENCRYPTION='n' fails with different error code
- grep_pattern replaced with search_pattern_in_file

MTR updated to look for plugin.defs in the plugin's mtr directory. It seems that upstream reorganized their test suites layout for commercial plugins and we may want to eventually follow them.

### CODE REVIEW

**Comment by gl-sergei** [comment]
https://ps80.cd.percona.com/job/percona-server-8.0-param/222/

File: plugin/keyring_vault/tests/mtr/temp_table_encrypt_keyring_vault.result [hunk 1]
```diff
@@ -6,18 +6,16 @@ call mtr.add_suppression("\\[Error\\] InnoDB: Encryption can't find master key,
...
 SET GLOBAL innodb_temp_tablespace_encrypt = ON;
 CREATE TEMPORARY TABLE t02 (a INT) ENGINE=InnoDB;
 CREATE TEMPORARY TABLE t03 (a TEXT) ENGINE=InnoDB;
 INSERT INTO t03 VALUES ('Curabitur laoreet, velit non interdum venenatis');
-Pattern not found.
 CREATE TEMPORARY TABLE t05 (a INT) ENGINE=InnoDB;
 INSERT INTO t05 VALUES (1), (2), (3);
 CREATE TEMPORARY TABLE t06 (a INT) ENGINE=InnoDB ENCRYPTION='n';
-ERROR HY000: InnoDB : ENCRYPTION is not accepted syntax for CREATE/ALTER table, for tables in general/shared tablespace.
+ERROR HY000: ENCRYPTION clause is not valid for temporary tablespace.
```
Comment (satya-bodapati) [inline plugin/keyring_vault/tests/mtr/temp_table_encrypt_keyring_vault.result:18]: Can we rename "tablespace" to "table" in error message. The statements are "CREATE TABLE" and not "CREATE TABLESPACE".

Please start a jenkins run and verify that all tests are covered by change in error message.

**Review by satya-bodapati — APPROVED** [review]
Ok to push after change in error message and after verifying jenkins results.

**Comment by gl-sergei** [comment]
as discussed on Slack we don't want to change the error message because it is upstream message. Instead bug report has been filed http://bugs.mysql.com/96093

**Pull request merged**